### ISSUE TITLE
Getting start eye catch

### ISSUE THREAD

**Issue by tombruijn** [appsignal/appsignal-docs#382]
We want to make the getting started guides more visible on the docs homepage. Now it's the first link on the index page: https://docs.appsignal.com/

But maybe we can do more to highlight that would be a first place to go if you're new to AppSignal.

We talked before about using some kind of "blocks" on the docs homepage to grab people's attention, maybe we can start using those for this?

**Comment by tombruijn** [comment]
@jonasbrinkhoff can you think of a component to reach this goal? Feel free to ping me or Shairyar if you need more input.

**Comment by tombruijn** [comment]
I've personally added some links on feature pages to relevant guides, like in the screenshot below:

![2020-10-22 16-57-32 - Deploy markers  AppSignal documentation](https://user-images.githubusercontent.com/282402/96890266-c5680080-1487-11eb-9612-b64d35f42873.png)

But I think we were thinking about something bigger, like how stripe does it, and with a button? See https://stripe.com/docs

**Comment by jonasbrinkhoff** [comment]
I was already working on something like a while back when I did the documentation redesign. I the end I didn't have enough time to add this feature, but the designs are already made: https://www.figma.com/file/JTJerpdO1zzihdVBympX6c/Documentation?node-id=124%3A610

Is this how you imagined the component?

I think my biggest issue was that these kind of blocks are not supported by default markdown. I started looking into creating custom markdown tags, but then ran out of time. If you want to pick this up, I'm happy to help with designs :)

**Comment by tombruijn** [comment]
@jonasbrinkhoff yes, something like that. I would maybe add something that incites to click it a bit more, like a button that says "Get started" or "Get to configuring", etc.

> I think my biggest issue was that these kind of blocks are not supported by default markdown. I started looking into creating custom markdown tags, but then ran out of time. If you want to pick this up, I'm happy to help with designs :)

We don't need to create a custom markdown tag for this while we are setting it up for the first time. Or at all. Depends how much we will use it.

We can use plain HTML for now. Markdown knows it should render the HTML as HTML normally.

**Comment by jonasbrinkhoff** [comment]
> We don't need to create a custom markdown tag for this while we are setting it up for the first time. Or at all. Depends how much we will use it.
> 
> We can use plain HTML for now. Markdown knows it should render the HTML as HTML normally.

Sweet! In that case, can you simply write down what kind of variations you need of that box (with button, with image, with icon, etc.) and I'll quickly create some designs :)

**Comment by tombruijn** [comment]
I would go for this one, where you can customize the arrow to be an emoji/fontawesome icon:

![image](https://user-images.githubusercontent.com/282402/96995454-8b056e80-152e-11eb-9f01-9fcf85a38810.png)

And one without the icon:

![image](https://user-images.githubusercontent.com/282402/96995542-a8d2d380-152e-11eb-9def-0aa7d5c64f6f.png)


I think we can put a button in there ourselves with tailwind, right?

**Comment by jonasbrinkhoff** [comment]
> I would go for this one, where you can customize the arrow to be an emoji/fontawesome icon:
> 
> And one without the icon:

Great! I guess then you don't need any designs from me? 🙃

> I think we can put a button in there ourselves with tailwind, right?

Yes!

**Comment by tombruijn** [comment]
> Great! I guess then you don't need any designs from me? 🙃

No? But is the component available in the design system?

**Comment by jonasbrinkhoff** [comment]
> No? But is the component available in the design system?

Nope, no component (yet)

**Comment by tombruijn** [comment]
Ok. Can you make one, or is that not on the planning?
